ksql-common/src/main/java/io/confluent/ksql/schema/ksql/FormatOptions.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.schema.ksql;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.function.Predicate;
+
+public final class FormatOptions {
```

Comment:
Can we use a more descriptive name, like `SchemaSqlFormatOptions` ?

Reply:
My preference is to not have really long names. This `schema` and `sql` is actually already captured in the package name.  I think it unnecessary to repeat in the class name, unless we're just trying to avoid a clash with some other class.